src/main/java/com/osiris/autoplug/client/Main.java

```diff
@@ -305,4 +368,57 @@ public static void main(String[] _args) {
         }
     }
 
+    public static class SSHManager {
```

Comment:
Make it it's own file and also add the console output reader to it. There shouldn't be anything in main except SSHManager.start() if enabled.